esp/esp/django_settings.py

```diff
@@ -290,7 +290,7 @@
 
 CONTACTFORM_EMAIL_CHOICES = (
     ('esp','Unknown'),
-    ('general','General ESP'),
+    ('general','General Program'),
```

Comment:
You could also just leave it at "General".

Reply:
Works for me!